packages/cli/templates/vue3/src/provideFieldPlugin.ts

```diff
@@ -0,0 +1,21 @@
+import { ref, provide } from 'vue'
+import { createFieldPlugin, FieldPluginResponse } from '@storyblok/field-plugin'
+
+let provided = false
+
+export const provideFieldPlugin = () => {
+  if (provided) {
+    throw new Error('Do not call `provideFieldPlugin()` more than once.')
+  }
+
+  provided = true
+  const plugin = ref<FieldPluginResponse>({
```

Comment:
If we use [reactive](https://vuejs.org/api/reactivity-core.html#reactive), the api consumers do not need to type `.value`.

Reply:
good idea